src/server/pegasus_server_impl.h

```diff
@@ -328,6 +329,11 @@ class pegasus_server_impl : public pegasus_read_service
 
     void update_user_specified_compaction(const std::map<std::string, std::string> &envs);
 
+    void update_rocksdb_dynamic_options(const std::map<std::string, std::string> &envs);
+
+    void
+    update_rocksdb_options_before_create_replica(const std::map<std::string, std::string> &envs);
```

Comment:
I suggest that rename this function as `set_rocksdb_options_before_creating`. `Updating` usually means that you update an old value into a new one, however, your caller function shows that there are not old value existed, it actually an seting operation.